cmd/podman/common/create.go

```diff
@@ -15,6 +15,16 @@ const sizeWithUnitFormat = "(format: `<number>[<unit>]`, where unit = b (bytes),
 
 var containerConfig = registry.PodmanConfig()
 
+// MapOptions takes the Container and Pod Create options, assigning the matching values back to podCreate for the purpose of the libpod API
+func MapOptions(containerCreate *entities.ContainerCreateOptions, podCreate *entities.PodCreateOptions) error {
```

Comment:
Can you improve the comment here to describe what needs to happen for this to work (IE, matching JSON tags on both sides)?

Reply:
sure